web/hooks/use-follows.ts

```diff
@@ -31,9 +31,11 @@ export const useFollowers = (userId: string | undefined) => {
   return followerIds
 }
 
-export const useContractFollows = (contractId: string) => {
+export const useContractFollows = (contractId: string | undefined) => {
   const [followIds, setFollowIds] = useState<string[] | undefined>()
-
+  if (!contractId) {
+    return
+  }
   useEffect(() => {
     return listenForContractFollows(contractId, setFollowIds)
   }, [contractId])
```

Comment:
Hmm, the early return would violate one of the rules of hooks: every code path must evaluate all the hooks.

https://reactjs.org/docs/hooks-rules.html

I would instead add an `if` within the `useEffect` to make sure `contractId` is defined.